src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/ZipArchiveExTests.cs

```diff
@@ -102,5 +142,50 @@ public void AddDirectoryCurrentDirectory ()
 				Directory.SetCurrentDirectory (cwd);
 			}
 		}
+
+		[Test]
+		//[Retry (50)]
```

Comment:
Comment could be removed here?

Otherwise this looks good to me, except the builds are hitting some issue on DevOps: `fatal: reference is not a tree: 2d6332c056206000a4751081c8cd7f462d9d9715`

Reply:
I'll remove the comment, and hopefully the next build will be good.